jupyterlab_server/config.py

```diff
@@ -163,6 +163,8 @@ def get_page_config(
             extension["mimeExtension"] = extbuild["mimeExtension"]
         if "style" in extbuild:
             extension["style"] = extbuild["style"]
+        if "version" in extbuild and "entrypoint" in extbuild and extbuild["version"] == "2":
+            extension["entrypoints"] = extbuild["entrypoint"]
```

Comment:
You could simplify that:

```suggestion
        # FIXME @experimental for plugin with no-code entrypoints.
        extension["entrypoints"] = extbuild.get("entrypoints")
```

The results will be `extension["entrypoints"] is None` if it is missing in the package.json file.

> I propose to use `entrypoints` plural too in the package.json instead of singular.